climatedata_api/map.py

```diff
@@ -1,11 +1,13 @@
+from datetime import datetime
 import json
 
 from flask import Flask, Response
 from flask import current_app as app
 from flask import request, send_file
 from werkzeug.exceptions import BadRequestKeyError
 
-from climatedata_api.utils import open_dataset, open_dataset_by_path, decode_compressed_points
+from climatedata_api.utils import open_dataset, open_dataset_by_path, decode_compressed_points, \
+    load_s2d_datasets_by_periods
```

Comment:
On pourrait le mettre sur plusieurs lignes:

```suggestion
from climatedata_api.utils import (
    open_dataset,
    open_dataset_by_path,
    decode_compressed_points,
    load_s2d_datasets_by_periods,
)
```